dtfs-central-api/api-tests/v1/tfm/amendments/tfm-amendments-get.api-test.js

```diff
@@ -173,7 +173,7 @@ describe('GET TFM amendments', () => {
 
       await api.put({ dealType: CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS, dealId }).to('/v1/tfm/deals/submit');
 
-      const { status, body } = await api.get(`/v1/tfm/facilities/${facilityId}/amendment/status/in-progress`);
+      const { status, body } = await api.get(`/v1/tfm/facilities/${facilityId}/amendments?status=in-progress`);
```

Comment:
Should the path not stay as it is with separable routes?
My understanding is that introducing query parameters is not consistent wit other API calls across the system.

Reply:
For this do you mean for all endpoints that now have query parameters? Might get @IgnatG 's input here for this question as my impression from the ticket is to use query parameters!